README.md

```diff
@@ -1 +1,6 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+### Hello microverse is my first project at Microverse. It is a basic project to explain how to create branches and Pull requests.
+
+## Built With
+### the project is built with Html and CSS as well as javascipt
+
```

Comment:
- [ ] I would suggest you please add a descriptive README file to your project. please use [this template](https://github.com/microverseinc/readme-template) and make sure that every heading should be present on your README file, the only optional is FAQ. And If you decide not to deploy the app please remove the live demo. Please write 1,2 sentences about all heading in your README file.